platform/features/hyperlink/bundle.js

```diff
@@ -0,0 +1,109 @@
+define([
+    './src/HyperlinkController',
+    'legacyRegistry'
+], function (
+    HyperlinkController,
+    legacyRegistry
+) {
+    legacyRegistry.register("platform/features/hyperlink", {
+    "name": "Hyperlink",
+    "description": "Insert a hyperlink to reference a link",
+    "extensions": {
+        "types": [
+            {
+                "key": "example.hyperlink",
+                "name": "Hyperlink",
+                "templateUrl": "templates/hyperlink.html",
+                "cssClass": "icon-page",
+                "description": "A hyperlink to redirect to a different link",
+                "features": ["creation"],
+                "properties": [
+                  {
+                    "key": "url",
+                    "name": "URL",
+                    "control": "textfield",
+                    "pattern": "^(ftp|https?)\\:\\/\\/",
+                    "required": true,
+                    "cssClass": "l-input-lg"
+                  },
+                  {
+                    "key": "displayText",
+                    "name": "Text to Display",
+                    "control": "textfield",
+                    "required": true,
+                    "cssClass": "l-input-lg"
+                  },
+                  {
+                    "key": "showTitle",
+                    "value": "false",
+                  },{
+                      "key": "displayFormat",
+                      "name": "Display Format",
+                      "control": "composite",
+                      "items": [
+                          {
+                              "control": "select",
+                              "options": [
+                                  {
+                                    "name": "Link",
+                                    "value":"link"
+                                  },
+                                  {
+                                      "value":"button",
+                                      "name": "Button"
+                                  }
+                              ],
+                              "cssClass": "l-inline"
+                          }
+                      ]
+                  },
+                  {
+                      "key": "openNewTab",
+                      "name": "Tab to Open Hyperlink",
+                      "control": "composite",
```

Comment:
No reason to wrap the select control in a composite control; this should just be a select control.